deploy/playbooks/roles/common/templates/prod.py.j2

```diff
@@ -0,0 +1,122 @@
+# {{ ansible_managed }}
+from prod_db import sqlalchemy
+from prod_jenkins import jenkins
+from prod_provider import provider
+
+# Server Specific Configurations
+server = {
+    'port': '8000',
+    'host': '0.0.0.0'
+}
+
+# Pecan Application Configurations
+app = {
+    'root': 'mita.controllers.root.RootController',
+    'modules': ['mita'],
+    'static_root': '%(confdir)s/public',
+    'template_path': '%(confdir)s/mita/templates',
+    'debug': False,
+}
+
+logging = {
+    'root': {'level': 'INFO', 'handlers': ['console']},
+    'loggers': {
+        'mita': {'level': 'DEBUG', 'handlers': ['console']},
+        'pecan': {'level': 'DEBUG', 'handlers': ['console']},
+        'py.warnings': {'handlers': ['console']},
+        '__force_dict__': True
+    },
+    'handlers': {
+        'console': {
+            'level': 'DEBUG',
+            'class': 'logging.StreamHandler',
+            'formatter': 'color'
+        }
+    },
+    'formatters': {
+        'simple': {
+            'format': ('%(asctime)s %(levelname)-5.5s [%(name)s]'
+                       '[%(threadName)s] %(message)s')
+        },
+        'color': {
+            '()': 'pecan.log.ColorFormatter',
+            'format': ('%(asctime)s [%(padded_color_levelname)s] [%(name)s]'
+                       '[%(threadName)s] %(message)s'),
+        '__force_dict__': True
+        }
+    }
+}
+
+from textwrap import dedent
+
+
+nodes = {
+    'precise': {
+        'script': dedent("""#!/bin/bash
+        curl -L "http://172.18.181.11/setup/slave/?token=727999862a610c7436318580de6f3e27&executors=1&labels=amd64+precise+trusty+wheezy+jessie+x86_64&nodename=precise__%s" | bash
+        """),
+        'keyname': 'jenkins-build',
+        'image_name': 'Ubuntu Trusty (2014.04) cloudimg',
+        'size': 'c3.mxlarge',
+        'labels': ['amd64', 'x86_64', 'precise', 'trusty', 'wheezy', 'jessie'],
+        'provider': 'openstack'
+    },
+    'wheezy': {
+        'script': dedent("""#!/bin/bash
+        curl -L "http://172.18.181.11/setup/slave/?token=727999862a610c7436318580de6f3e27&executors=1&labels=amd64+precise+trusty+wheezy+jessie+x86_64&nodename=wheezy__%s" | bash
```

Comment:
tokens should be private (not pushed here) so maybe a variable is needed here as well